src/components/EarlyAccessRegistrySurvey.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import DynamicSurvey from "./DynamicSurvey";
+import { Survey } from "posthog-js";
+import { usePostHog } from "posthog-js/react";
+
+// TODO: Make this an ENV variable
+const SURVEY_ID = "019683f6-4fe2-0000-d182-6ef8f3982fc3";
+
+export const EarlyAccessRegistrySurvey = () => {
+  const posthog = usePostHog();
+  const [surveyData, setSurveyData] = useState<Survey | null>(null);
+
+  function loadSurvey() {
+    posthog.getSurveys((surveys) => {
+      const survey = surveys.find((s) => s.id === SURVEY_ID);
+
+      if (survey) {
+        setSurveyData(survey);
+        posthog?.capture("Early Access Registry Survey viewed", {
+          survey_version: survey.id,
+        });
+      }
+    });
+  }
+
+  useEffect(() => {
+    if (posthog) {
+      loadSurvey();
+    } else {
+      console.warn("PostHog is not initialized");
+    }
```

Comment:
```suggestion
    if (!posthog) return;
    posthog.onFeatureFlags(loadSurvey);
```

So, the problem is that this renders on the server the first time, and Posthog is undefined. We can fix this in different ways: with this callback, setTimeout, queueMicrotask